### PR TITLE
[bitnami/mariadb] Change database name for consistency purposes

### PR DESCRIPTION
**Pull request by ebCrypto**
Signed-off-by: ebCrypto <44279886+ebCrypto@users.noreply.github.com>

### Applicable issues

  - fixes #

### Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [ ] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [ ] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/mariadb/README.md [hunk 1]
```diff
@@ -553,4 +553,4 @@ Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
-limitations under the License.
\ No newline at end of file
+limitations under the License.
```
Comment (aoterolorenzo) [inline bitnami/mariadb/README.md:556]: Hi @ebCrypto,

Could you revert this line break change?

Comment (ebCrypto) [reply bitnami/mariadb/README.md:556]: If you `Display the rich diff`
there is only one code change related to the database base:

image

Comment (aoterolorenzo) [reply bitnami/mariadb/README.md:556]: Seems the rich this omits this kind of changes. However, I'm afraid your IDE is still adding a new line at the end of the file. As this as it is an unrelated change and also for consistency purposes we would need to remove it.

```
lorenzoa@lorenzoa-a01 charts-ebCrypto % git diff HEAD^
diff --git a/bitnami/mariadb/README.md b/bitnami/mariadb/README.md
index c719765af..cfc7e8bd3 100644
--- a/bitnami/mariadb/README.md
+++ b/bitnami/mariadb/README.md
.
.
.
.
-limitations under the License.
\ No newline at end of file
+limitations under the License.

```

Comment (ebCrypto) [reply bitnami/mariadb/README.md:556]: Ha I understand what you are saying now. It makes sense. Yet, I made the update directly on GitHub.com so it seems like bitnami can't support that. At this point it probably makes sense for you to close this PR and correct the documentation with a compatible IDE.

Comment (aoterolorenzo) [reply bitnami/mariadb/README.md:556]: May I could interact in your fork and branch to correct it? You will still have the ownership of the changes in this way.

**Pull request closed without merging**